src/RecoveryServices/RecoveryServices/ChangeLog.md

```diff
@@ -18,6 +18,12 @@
         - Additional information about change #1
 -->
 ## Upcoming Release
+* modified policy validation limits as per backup service.
+* Added Zone Redundancy for Recovery Service Vaults.
+* Azure Site Recovery support to Proximity placement group and Availability zone as optional input parameters at enabling protection for Vmware to Azure.
+* Azure Site Recovery support to Proximity placement group and Availability zone as optional input parameters at enabling protection for HyperV to Azure.
+* Azure Site Recovery support to update replication protected item with Proximity placement group and Availability zone for Vmware to Azure.
+* Azure Site Recovery support to update replication protected item with Proximity placement group and Availability zone for HyperV to Azure. 
```

Comment:
Could you simplify the changlog messages to 1~2 lines because we will merge all module's changlog msg into one Az.psd1, which has maximum character limit.

Reply:
> Could you simplify the changlog msgs to 1~2 lines because we will merge all module's changlog msgs into one Az.psd1, which has maximum character limit.

Done